.github/container/build-jax.sh

```diff
@@ -262,10 +266,8 @@ time python "${SRC_PATH_JAX}/build/build.py" \
     --enable_cuda \
     --build_gpu_plugin \
     --gpu_plugin_cuda_version=$TF_CUDA_MAJOR_VERSION \
-    --cuda_path=$TF_CUDA_PATHS \
-    --cudnn_path=$TF_CUDNN_PATHS \
-    --cuda_version=$TF_CUDA_VERSION \
-    --cudnn_version=$TF_CUDNN_VERSION \
+    --bazel_options=--repo_env=HERMETIC_CUDA_VERSION=$CUDA_VERSION \
```

Comment:
```suggestion
    --bazel_options=--repo_env=HERMETIC_CUDA_VERSION=$TF_CUDA_VERSION \
```

Reply:
Sadly this won't work because `TF_CUDA_VERSION` will be `12.5`, the `CUDA_VERSION` from container env is `12.5.0` and `HERMETIC_CUDA_VERSION` requires to be a full version.